portal/v2/src/ClusterList.tsx

```diff
@@ -431,8 +505,10 @@ export function ClusterList(props: {
         styles={controlStyles}
       />
       <Separator styles={separatorStyle} />
-
```

Comment:
Makes sense to keen this line break IMO, because we do between all other components.